xseed.sh

```diff
@@ -8,7 +8,7 @@
 
 # Load environment variables from .env file if it exists
 # in the same directory as this bash script
-VERSION='3.1.0'
+VERSION='3.2.0'
```

Comment:
```suggestion
VERSION='3.1.0'
```
I'm not sure how @bakerboy448 feels, but I would say that given he generally bumps versions in their own commits prior to publishing a release, you should revert this and let him manage the versions bumps.